src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int height;
+    private int middleLine;
+
+    public IsoscelesTrapezoid(int height, int middleLine, String color) {
+        super(color);
+        this.height = height;
+        this.middleLine = middleLine;
+    }
+
+    @Override
+    public void draw() {
+    }
```

Comment:
The draw() method is empty. To implement drawing the figure, you should add a meaningful implementation, like printing out the figure properties and its area.